src/twilio/twilio_types.bal

```diff
@@ -42,6 +42,51 @@ public type SmsResponse record {
     string priceUnit = "";
 };
 
+# Represents Twilio WhatsApp message response. More details of the message format is
+# accessible from https://www.twilio.com/docs/sms/api/message-resource#create-a-message-resource
+# + sid - Unique identifier of the account
+# + dateCreated - The date and time that this resource was created
+# + dateUpdated - The date and time that this resource was last updated
+# + dateSent - The date and time where the outgoing message was sent
+# + accountSid - The unique identifier of the account which sent the message
+# + toNumber - The phone number to which the message was sent
+# + fromNumber - The phone number from which the message was sent
+# + messageServiceSid - The SID of the Messaging Service where we want to associate with the message
+# + body - The text of the message we want to send
+# + status - Status of the voice call (queued, failed, sent, delivered, undelivered)
+# + numSegments - The number of segments which make up the complete message.
+# + numMedia - The number of media files associated 
+# + direction - The direction of the message (inbound, outbound-api, outbound-call, outbound-reply).
+# + apiVersion - The API version used to process the message.
+# + price - The price amount of the SMS. This is set to null for WhatsApp messages.
+# + priceUnit - The price currency. This is set to null for WhatsApp messages.
+# + errorCode - The error code returned if the message status is failed or undelivered.
+# + errorMessage - The description of the error_code if the message status is failed or undelivered.
+# + uri - The URI of the resource, relative to https://api.twilio.com
+# + subresourceUris - A list of related resources identified by their URIs relative to https://api.twilio.com 
+public type ProgrammableSMSResponse record {
```

Comment:
Shall we rename the record name as follows in order to maintain the convention of record naming?
```suggestion
public type ProgrammableSmsResponse record {
```

Reply:
Based on the offline discussions we decided to refactor the code by creating a separate type for WhatsApp messages called WhatsAppResponse. This removed the requirement of having a type called ProgrammableSmsResponse. Furthermore, we keep the SmsResponse record type as it is, but we add all the remaining fields which were not present in the previous code.